api/handlers/datasources.py

```diff
@@ -32,6 +34,7 @@ async def create_datasource(
 
 
 @router.get("/api/datasources", response_model=list[DataSource])
+@requires("IQEngine-User")
```

Comment:
How is this going to support the current capabilities of anonymous access on regular IQEngine?

Reply:
@PaulDFoster we will need this in before merging as this will stoo endpoints from working on staging and main iqengine.org site